prowler/providers/aws/services/rds/rds_instance_deprecated_engine_version/rds_instance_deprecated_engine_version.py

```diff
@@ -0,0 +1,30 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.rds.rds_client import rds_client
+
+
+class rds_instance_deprecated_engine_version(Check):
+    def execute(self):
+        findings = []
+        available_versions = []
+
+        for instance in rds_client.db_instances:
+            available_versions.clear()
+            ##Filling the list of the available versions of the current engine
+            for iterate_version in rds_client.db_engines:
```

Comment:
What is the rationale to set a `PASS`? What is a valid engine?

Reply:
The db_engines contains the info of the engine versions that are currently available for each engine so if the engine version of the instance is in the db_engines it has a valid version.